docs/tutorial/notebook/03_train_model.ipynb

```diff
@@ -0,0 +1,583 @@
+{
```

Comment:
<p>Also talk about:</p><p>Continuing training from a checkpoint.</p><p>Point to the tensorboard tutorial for visualizing training progress.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D-ML/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>